components/centos/centos-docker-base-setup/README.adoc

```diff
@@ -0,0 +1,37 @@
+= Vagrantfile: CentOS Docker Base
+:toc:
```

Comment:
@budhrg even in files that have one one 2heading, the TOC is useful because when we publish the docs it will be translated to a sidebar entry in the TOC tree.